src/ast/irbuilderbpf.cpp

```diff
@@ -450,6 +480,116 @@ Value *IRBuilderBPF::CreateMapLookupElem(Value *ctx,
   return ret;
 }
 
+Value *IRBuilderBPF::CreatePerCpuMapSumElems(Value *ctx,
```

Comment:
This implements a non-trivial algorithm. Could we perhaps include a pseudo-code in the comment to make it clear what's the intention to do here?